### PR TITLE
[Windows] Implement Border properties in RadioButtonHandler

### PR DESCRIPTION
**Pull request by jsuarezruiz**
### Description of Change ###

Implement Border properties in RadioButtonHandler on Windows.

- BorderColor
- BorderWidth
- CornerRadius

### PR Checklist ###

- [x] Targets the correct branch 
- [ ] Tests are passing (or failures are unrelated)

#### Does this PR touch anything that might affect accessibility?
- No

### CODE REVIEW

File: src/Core/src/Platform/Windows/RadioButtonExtensions.cs [hunk 1]
```diff
@@ -47,6 +48,21 @@ public static void UpdateContent(this RadioButton nativeRadioButton, IRadioButto
...
-			
+
+		public static void UpdateStrokeColor(this RadioButton nativeRadioButton, IRadioButton radioButton)
+		{
+			nativeRadioButton.BorderBrush = radioButton.StrokeColor != null ? radioButton.StrokeColor.ToNative() : (WBrush)UI.Xaml.Application.Current.Resources["ButtonBorderThemeBrush"];
+		}
+
+		public static void UpdateStrokeThickness(this RadioButton nativeRadioButton, IRadioButton radioButton)
+		{
+			nativeRadioButton.BorderThickness = radioButton.StrokeThickness == -1 ? WinUIHelpers.CreateThickness(3) : WinUIHelpers.CreateThickness(radioButton.StrokeThickness);
```
Comment (Redth) [inline src/Core/src/Platform/Windows/RadioButtonExtensions.cs:59]:
```suggestion
			nativeRadioButton.BorderThickness = radioButton.StrokeThickness == <= 0 ? WinUIHelpers.CreateThickness(0) : WinUIHelpers.CreateThickness(radioButton.StrokeThickness);
```

Setting this to 3 by default was causing the border to always show up unless explicitly set to 0.

File: src/Core/src/Platform/Windows/MauiRadioButton.cs [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+			if (_contentPresenter != null)
+				_contentPresenter.Background = BackgroundColor;
+			Background = new UI.Xaml.Media.SolidColorBrush(UI.Colors.Transparent);
+		}
+
+		void UpdateBorderRadius()
+		{
+			if (_contentPresenter != null)
+				_contentPresenter.CornerRadius = WinUIHelpers.CreateCornerRadius(BorderRadius);
```
Comment (Redth) [inline src/Core/src/Platform/Windows/MauiRadioButton.cs:75]: In this method, we potentially pass -1 which causes an exception at runtime, so perhaps change the `WinUIHelpers.CreateCornerRadius` method to check for `if (all <=0) return new WCornerRadius();` to ensure an empty radius.

File: src/Core/src/Platform/Windows/MauiRadioButton.cs [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+			((MauiRadioButton)d).UpdateBorderRadius();
+		}
+
+		void UpdateBackgroundColor()
+		{
+			if (BackgroundColor == null)
+				BackgroundColor = Background;
+
+			if (_contentPresenter != null)
+				_contentPresenter.Background = BackgroundColor;
```
Comment (Redth) [inline src/Core/src/Platform/Windows/MauiRadioButton.cs:68]: Seems we need to keep track of the focused visual state, as when you hover over the radiobutton, the border disappears.

Comment (jsuarezruiz) [reply src/Core/src/Platform/Windows/MauiRadioButton.cs:68]: Applied changes to fix the wrong behavior with the mouse hover state.

**Comment by jsuarezruiz** [comment]
Applied more changes to cover the Border behavior with all the different visual states.
radiobutton-border-windows

**Review by Redth — APPROVED** [review]

**Pull request merged**